### PR TITLE
Replace GS ssh key with keys from Values file

### PR DESCRIPTION
**Pull request by alex-karpeichyk**
This PR:

- Changes the sshFiles helper to fetch trusted keys from Values.yaml
- Deletes trusted-user-ca-keys.pem

### Testing

- [ ] Fresh install works.
- [ ] Upgrade from previous version works.

### Checklist

- [x] Update changelog in `CHANGELOG.md`.
- [x] Make sure `values.yaml` is valid.

### CODE REVIEW

**Review by eugene-eeo — APPROVED** [review]

**Comment by erkanerol** [comment]
@alex-karpeichyk can you mimic this interface? https://github.com/giantswarm/cluster-cloud-director/pull/82/files#diff-6e53353f188ce9fada1c96ca868b379c066de1c6e295d6d0b115a55d8539ebb6R96

**Comment by alex-karpeichyk** [comment]
@erkanerol Made the change, if you could take a look please?

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Add `items` schema for `.controlPlane.availabilityZones` in `values.schema.json`.
 
+## [0.17.1] - 2023-02-10
```
Comment (erkanerol) [inline CHANGELOG.md:14]: please add this under `unrelased` section. that is updated with versions by ci jobs.

File: helm/cluster-openstack/Chart.yaml [hunk 2]
```diff
@@ -3,7 +3,7 @@ name: cluster-openstack
 description: A helm chart for creating Cluster API clusters with the OpenStack infrastructure provider (CAPO).
 home: https://github.com/giantswarm/cluster-openstack
 type: application
-version: 0.17.0
+version: 0.17.1
```
Comment (erkanerol) [inline helm/cluster-openstack/Chart.yaml:6]: don't update manually. will be updated by ci jobs.

File: CHANGELOG.md (hunk 1 above)
Comment (kamal-pandala-thg) [reply CHANGELOG.md]: Moved it under unreleased section.

File: helm/cluster-openstack/Chart.yaml (hunk 2 above)
Comment (kamal-pandala-thg) [reply helm/cluster-openstack/Chart.yaml]: Remove the manual update.

**Review by erkanerol — APPROVED** [review]

**Comment by eugene-eeo** [comment]
hey @erkanerol, thanks for merging this in. is it possible to get these changes tagged and released as well?

**Comment by erkanerol** [comment]
@eugene-eeo Here https://github.com/giantswarm/cluster-openstack/pull/132

**Pull request merged**